src/index.html

```diff
@@ -2,11 +2,491 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no,   initial-scale=1.0, maximum-scale=1.0,   minimum-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.googleapis.com"
+    >
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="attr-req-value"
+    >
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Serif:opsz,wght@8..144,400;8..144,500&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet"
+    >
   </head>
-  <body>
-    <h1>Catalog</h1>
+
+  <body class="body">
+    <header class="header">
+      <a class= "header__logo" href="#home">
+        <img src="./images/logo.png" alt="Moyo logo">
+      </a> 
+      
+      <nav class="nav">
+        <ul class="nav__navigation">
+          <li class="nav__list">
+            <a class="nav__link is-active" href="#apple">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a class= "nav__link" href="#samsung">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a class= "nav__link" href="#smartphones">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a
+              class= "nav__link"
+              data-qa="nav-hover" 
+              href="#laptops&computers"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a class= "nav__link" href="#gadgets">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a class= "nav__link" href="#tablets">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a class= "nav__link" href="#photo">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a class= "nav__link" href="#video">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__content content">
+        <div data-qa="card" 
+             class="content__product card"
+        >
+          <img 
+            class="card__image" 
+            src="./images/imac.jpeg" 
+            alt="Apple iMac"
+          >
+
+          <h1 class="card__content">
```

Comment:
h1 can be only one per page. 
```suggestion
          <h2 class="card__content">
```